mirror/src/main/java/com/bakdata/quick/mirror/range/padder/ZeroPadder.java

```diff
@@ -25,4 +25,6 @@
     String padZero(final T number);
 
     Class<T> getPadderClass();
+
+    T convertStringToNumber(final String stringValue);
```

Comment:
The method name should indicate that this handles the inclusive/exclusive logic:
```suggestion
    T getEndOfRange(final String stringValue);
```

Reply:
Also, I think JavaDocs for all methods here would be helpful